### PR TITLE
2199 remove unnecessary odc pics

### PR DESCRIPTION
**Pull request by ghost**
https://issues.redhat.com/browse/RHDEVDOCS-2199

- [x] Reviewed by <QE/SME>
- [x] peer-reviewed by DT docs 
- [x] peer-reviewed by ocp docs
- merge to ?

Reasons to remove the picture:
- illegible, even in FHD, even zoomed
- breaks the text
- info is present in the text

### CODE REVIEW

**Review by Preeticp — APPROVED** [review]
/lgtm 
I agree with the reasons mentioned by you:
- it is huge
- it is illegible, even in FHD resolution.
- it is a form, user can see it
As well as in line with keeping screenshots to the minimal on account of maintenance and adding them only where an end user is likely to get confused let's remove this screenshot.

**Review by rohitkrai03 — APPROVED** [review]
LGTM

**Review by pneedle-rh — COMMENTED** [review]
@pmacko1 I have now completed my review. Please let me know if you have any questions about my suggestions.

I have added labels for enterprise-4.3, enterprise-4.4, enterprise-4.5 and enterprise-4.6 as I can see that the image is present for each of those versions of the doc.

File: modules/odc-importing-codebase-from-git-to-create-application.adoc [hunk 1]
```diff
@@ -14,8 +14,6 @@ The following procedure walks you through the *From Git* option in the *Develope
 . In the *Add* view, click *From Git* to see the *Import from git* form.
 +
 .Import from Git
```
[the remark this answers is not in the document]
Comment (pmacko1) [reply modules/odc-importing-codebase-from-git-to-create-application.adoc:16]:
```suggestion
```

**Comment by pmacko1** [comment]
squashed

File: modules/odc-importing-codebase-from-git-to-create-application.adoc [hunk 2]
```diff
@@ -13,9 +13,6 @@ The following procedure walks you through the *From Git* option in the *Develope
 
 . In the *Add* view, click *From Git* to see the *Import from git* form.
 +
```
Comment (pneedle-rh) [inline modules/odc-importing-codebase-from-git-to-create-application.adoc:15]: @pmacko1 can you please also remove this plus sign? Sorry I should have spotted that during my first review.

Comment (pmacko1) [reply modules/odc-importing-codebase-from-git-to-create-application.adoc:15]: yeah sure, I didn't notice it as well.
```suggestion
```

Comment (pmacko1) [reply modules/odc-importing-codebase-from-git-to-create-application.adoc]: i love this new github feature, btw :)
ctrl+g or <triplebacktick>suggestion
<suggestion>
<triplebacktick>

**Comment by pmacko1** [comment]
fixed and squashed @pneedle-rh

**Pull request merged**